src/Components/Shared/Appointments/AddAppointment/index.js

```diff
@@ -10,46 +10,60 @@ import Success from './Success';
 import { useHistory } from 'react-router-dom';
 import Loading from '../../../../Practitioner/Shared/CardLoading';
 
-const initalState = {
-    category: "",
-    time: null,
-    note: "",
-    datetime: DateTime.local().toISO(),
-    tempDatetime: DateTime.local().toISO()
-}
+const initialState = {
+  category: '',
+  time: null,
+  note: '',
+  datetime: DateTime.local().toISO(),
+  tempDatetime: DateTime.local().toISO(),
+};
 
 export default function AddAppointment({ patientId }) {
+  const { t } = useTranslation('translation');
+  const history = useHistory();
 
-    const { t } = useTranslation('translation');
-    const history = useHistory();
-
-    const [state, setState] = useState({
-        ...initalState
-    })
-
-    const { execute, status, value, error, reset } = useAsync({
-        asyncFunc: SharedAPI.addAppointment,
-        immediate: false,
-        funcParams: [patientId, state]
-    })
+  const [state, setState] = useState({
+    ...initialState,
+  });
 
-    const resetState = () => {
-        setState({ ...initalState })
-        reset();
+  const { execute, status, value, error, reset } = useAsync({
+    asyncFunc: SharedAPI.addAppointment,
+    immediate: false,
+    funcParams: [patientId, state],
+  });
 
-    }
-
-    return (
+  const resetState = () => {
+    setState({ ...initialState });
+    reset();
+  };
 
+  return (
+    <>
+      <TopPageLabel sticky title={t('appointments.addAppointment')} />
+      {status === 'idle' && (
+        <Form state={state} setState={setState} handleSubmit={execute} />
+      )}
+      {status === 'loading' && (
         <>
-            <TopPageLabel sticky title={t('appointments.addAppointment')} />
-            {status === "idle" && <Form state={state} setState={setState} handleSubmit={execute} />}
-            {status === "loading" && <>
-                <Box height="3rem" />
-                <Loading />
-            </>}
-            {status === "success" && <Success handleExit={history.goBack} handleReset={resetState} />}
```

Comment:
i think that this component was one that I accidentally reformatted so it looks like I changed it. I don't recall touching this component in ways that changed any core functionality because this was the old way to add an appointment.